data/language/en-GB.yml

```diff
@@ -2350,3 +2350,15 @@ strings:
   2295: "Expert"
   2296: "Vanilla"
   2297: "Custom"
+  2298: "Components"
+  2299: "Sort by year designed"
+  2300: "Sort by name"
+  2301: "Unpowered"
+  2302: "Powered"
+  2303: "Unlocked"
+  2304: "Locked"
+  2305: "Cargo"
+  2306: "All cargo types"
+  2307: "{MOVE_X 10}Supports {STRINGID} {SPRITE}"
```

Comment:
I think having a word (ie "Supports") repeated in the dropdown is a little repetitive, and its reasonably obvious to a player I think that this list is a filter, so my vote is to remove the "Supports" from the dropdown items

Reply:
Dropped the verb/prefix, so the dropdown now looks like this:

<img width="162" alt="Screenshot 2023-10-10 at 18 22 23" src="https://github.com/OpenLoco/OpenLoco/assets/604665/797af034-0068-4705-afb0-8c03245f7510">